cdhweb/pages/management/commands/exodus.py

```diff
@@ -95,15 +97,24 @@ def handle(self, *args, **options):
             )
         root = Page.objects.get(depth=1)
         root.add_child(instance=homepage)
-        root.save()
+        root.save(log_action=False)
+        exodize_history(old_homepage, homepage)
 
         # point the default site at the new homepage and delete any others.
         # if they are deleted prior to switching site.root_page, the site will
-        # also be deleted in a cascade, which we don't want
+        # also be deleted in a cascade, which we don't want.
         site = Site.objects.get()
         site.root_page = homepage
         site.save()
-        Page.objects.filter(depth=2).exclude(pk=homepage.pk).delete()
+        Page.objects.filter(depth=2) \
+                    .exclude(pk=homepage.pk) \
+                    .delete()
+
+        # also delete any log entries referencing pages from previous runs, 
+        # otherwise they'll stick around and point to nothing since PKs change.
+        # objects.delete() isn't implemented so we do it via iterating
+        for entry in PageLogEntry.objects.all():
```

Comment:
You should be able to just do `.all().delete()` (delete isn't implemented on the manager, but it is available on the queryset)

Reply:
💡 thank you!!